src/main/java/org/humancellatlas/ingest/project/Project.java

```diff
@@ -65,6 +65,15 @@ public SubmissionEnvelope getOpenSubmissionEnvelope(){
         return null;
     }
 
+    public Boolean getHasOpenSubmission(){
+        for (SubmissionEnvelope submissionEnvelope : this.submissionEnvelopes) {
+            if (submissionEnvelope.isOpen()){
```

Comment:
Does this work even with `lazy = true` on the submissionEnvelopes DBRef? Would expect it to return a null pointer exception, lazy-loaded refs should only have the ID property populated

Reply:
This does indeed work.
Lazy loaded refs populate the object, the moment it is first accessed.